doc/development.md

```diff
@@ -46,8 +46,7 @@ Before we can start you need to match some prerequirements on your host machine.
      #!/usr/bin/env bash
 
      export pre_onadmin_installcrowbar=$(base64 --wrap=0 <<'EOS'
-       zypper ar -f http://download.suse.de/ibs/Devel:/Cloud:/Shared:/Rubygem/SLE_11_SP3/Devel:Cloud:Shared:Rubygem.repo;
-       zypper mr -r -e sp3sdk;
+       zypper -n in http://dist.suse.de/install/SLP/SLE-12-SDK-GM/x86_64/DVD1/suse/x86_64/sqlite3-devel-3.8.3.1-1.14.x86_64.rpm
```

Comment:
yes I thought it would have just been this one package, but in fact there are more dependencies, so i will add the repo.